src/utils/sendEmail.js

```diff
@@ -0,0 +1,47 @@
+/*
+Author: chankruze (chankruze@gmail.com)
+Created: Wed Oct 05 2022 17:44:31 GMT+0530 (India Standard Time)
+
+Copyright (c) geekofia 2022 and beyond
+*/
+
+const mailgunClient = require("../lib/mailgun");
+
+if (!process.env.DOMAIN) {
+  throw new Error("Please set `DOMAIN` value in environment variable");
+}
+
+if (!process.env.MAILGUN_EMAIL_FROM_NAME) {
+  throw new Error(
+    "Please set `MAILGUN_EMAIL_FROM_NAME` value in environment variable"
+  );
+}
+
+if (!process.env.MAILGUN_EMAIL_TO) {
+  throw new Error(
+    "Please set `MAILGUN_EMAIL_FROM` value in environment variable"
+  );
+}
+
+const DOMAIN = process.env.DOMAIN;
+const MAILGUN_EMAIL_FROM_NAME = process.env.MAILGUN_EMAIL_FROM_NAME;
+const MAILGUN_EMAIL_FROM_TO = process.env.MAILGUN_EMAIL_FROM_TO;
```

Comment:
`MAILGUN_EMAIL_FROM_TO` is not specified as an .env variable. Did you mean `MAILGUN_EMAIL_TO` ?

Reply:
Ya, I figured out that, I forgot to ammend the commit locally. Fixing it now.